.gitignore

```
@@ -129,6 +129,7 @@ dmypy.json
 docs/build/
 docs/site/
 Manifest.toml
+Manifest-v*.toml
```

You could combine these:
```suggestion
Manifest*.toml
```

---

that would also allow other things 😉